.gitignore

```diff
@@ -1,3 +1,9 @@
 /node_modules
 /target
-
+/.shadow-cljs
+.lein-repl-history
+.nrepl-port
+/public
+.lsp
+/client/resources
+/common/src
```

Comment:
Instead of these two lines, can you add
```
/client/resources/manifest.edn
/client/resources/public/wh.css
/client/resources/public/wh.min.css
/client/resources/public/wh.css.map
/client/resources/public/public.js
/client/resources/public/public.js.map
/client/resources/public/wh-legacy.js
/client/resources/public/wh-legacy.js.map
/client/resources/public/wh-legacy.css
/client/resources/public/wh-legacy.css.map
/client/resources/public/public.css
/client/resources/public/public.css.map
/client/resources/public/wh-styles.css
/client/resources/public/wh-styles.css.map
/client/resources/public/public.min.js
/client/resources/public/public.min.js.map
/client/resources/public/wh-styles.js
/client/resources/public/wh-styles.js.map
/client/resources/public/wh.min.js
/client/resources/public/wh.min.js.map
```